CHANGELOG.md

```diff
@@ -3,6 +3,96 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v5.11.1
+
+_May 20, 2022_
+
+We'd like to offer a big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Support localization on the date and time picker components (#4517) @alexfauquette
+
+  Texts can be translated in the pickers components, similar to what can be done in the data grid component. Check the [documentation](https://mui.com/x/react-date-pickers/localization/) for more information.
+
+- 📃 Add support for column spanning when exporting to Excel (#4830) @cherniavskii
+
+  <img src="https://user-images.githubusercontent.com/13808724/167691417-bf6baeb9-d409-4134-acb6-aadaf6434de9.png" width="800">
+
+- 🐞 Bugs fixes
+
+### `@mui/x-data-grid@v5.11.1` / `@mui/x-data-grid-pro@v5.11.1` / `@mui/x-data-grid-premium@v5.11.1`
+
+#### Changes
+
+- [DataGrid] Add a CSS class corresponding to current density (#4858) @m4theushw
+- [DataGrid] Execute the pipe-processors in their initialization order (#4913) @flaviendelangle
+- [DataGrid] Fix rendering of the no rows overlay when the `loading` prop is changed (#4910) @m4theushw
+- [DataGridPremium] Add `exceljs` to the dependencies (#4939) @alexfauquette
+- [DataGridPremium] Support column spanning in the Excel export (#4830) @cherniavskii
+- [l10n] Improve Russian (ru-RU) locale (#4864) @arvkonstantin
+
+### `@mui/x-date-pickers@5.0.0-alpha.4` / `@mui/x-date-pickers-pro@5.0.0-alpha.4`
+
+#### Breaking changes
+
+The following props have been removed: `clearable`, `showTodayButton`, `cancelText`, `okText`.
```

Comment:
- The props related to the action bar buttons have been removed (`clearable`, `showTodayButton`, `cancelText`, `okText`)

    To decide which button must be displayed and in which order, you can now use the `actions` prop of the `actionBar` component slot props.

    ```jsx
    <DatePicker
      // The actions will be the same between desktop and mobile
      componentsProps={{ actionBar: { actions: ['clear'] } }}

      // The actions will be the different between desktop and mobile
      componentsProps={{ actionBar: { actions: (variant) => variant === 'desktop' ? [] : ['clear'] } }}
    />
    ```

    The build-it `ActionBar` component supports 4 different actions: `'clear'`, `'cancel'`, `'accept'`, and `'today'`.
    The default values are `['cancel', 'accept']` on mobile and `[]` on desktop.

    If you need other actions, you can provide your own component to the `ActionBar` component slot

    ```jsx
    <DatePicker
      componentsProps={{ ActionBar: CustomActionBar }}
    />
    ```

Reply:
Yes, showing the two ways to use the `actions` prop is a better idea than explaining it.

I don't see ways to improve the structure. I only found two typos

```diff
-// The actions will be the different between desktop and mobile
+// The actions will be different between desktop and mobile
```

```diff
-The build-it `ActionBar`
+The built-in `ActionBar`
```

Reply:
Applied